### PR TITLE
improvement: assignStyledNodeCollection now also handles update

### PR DESCRIPTION
**Pull request by larsmoa**
Removes the need for a seperate updateStyledNodeCollection() function

### CODE REVIEW

File: viewer/core/src/public/migration/Cognite3DViewer.ts [hunk 1]
```diff
@@ -1572,34 +1572,38 @@ export class Cognite3DViewer {
...
+          renderController.clearNeedsRedraw();
+          this.revealManager.resetRedraw();
+          this._clippingNeedsUpdate = false;
+          const renderTime = Date.now() - start;
+
+          this._events.sceneRendered.fire({ frameNumber, renderTime, renderer: this.renderer, camera: this.camera });
+        }
       }
+    } finally {
+      this.latestRequestId = requestAnimationFrame(this._boundAnimate);
```
Comment (haakonflatval-cognite) [inline viewer/core/src/public/migration/Cognite3DViewer.ts:1606]: What is the significance of moving this here? What is the purpose of `this.latestRequestId`?

**Review by haakonflatval-cognite — CHANGES_REQUESTED** [review]
Looks good, just a comment I'd like answers for

File: viewer/core/src/public/migration/Cognite3DViewer.ts (hunk 1 above)
Comment (larsmoa) [reply viewer/core/src/public/migration/Cognite3DViewer.ts:1606]: Oh, that should not be part of this PR - fixing.

**Review by haakonflatval-cognite — APPROVED** [review]
Looks alright now :+1:

**Pull request merged**